graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -567,9 +719,47 @@ class RenameSchemaTypesVisitor(Visitor):
     # named "Foo", suppressed_type_names will contain "Foo".
     suppressed_type_names: Set[str]
 
+    # reverse_field_name_map maps type name to a dict, which in turn maps the name of a field in the
+    # renamed schema to the name of the field in the original schema, if the field has different
+    # names in the original schema and the new schema. If field_renamings would rename a field named
+    # "foo" (in a type named "Baz") to "bar", then reverse_field_name_map["Baz"] will map "bar" to
+    # "foo".
+    reverse_field_name_map: DefaultDict[str, Dict[str, str]]
+
+    # Collects no-op renamings for fields, mapping the type name that contains the field to the set
+    # of field names for which field_renamings contained no-op renamings. Applies only to types for
+    # which field_renamings[type_name] is iterable. If field_renamings would rename a field named
+    # "foo" to "foo" (in a type named "Bar"), or if it attempts to rename a field named "foo" (in a
+    # type named "Bar") when such a field does not exist, no_op_field_renamings will map "Bar" to a
+    # set containing "foo".
+    no_op_field_renamings: DefaultDict[str, Set[str]]
+
+    # Collects type names for each object type that has field renamings and had them applied. After
```

Comment:
A concrete example: 
* Let the schema have a single type `Foo`
* Let `field_renamings` be a dict with field renamings for `Foo` and `Bar`

Then `types_with_field_renamings_processed` should contain the string `Foo` because its field renamings were applied. Then, when detecting no-op renamings, we have `set(field_renamings) - visitor.types_with_field_renamings_processed` which is `{"Bar"}`.